setup.py

```diff
@@ -17,6 +17,19 @@
 with open(os.path.join(here, 'sewer', '__version__.py'), 'r') as f:
     exec(f.read(), about)
 
+dns_provider_deps_map = {
+    'aliyun': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
+    'alicloud': ['aliyun-python-sdk-core-v3', 'aliyun-python-sdk-alidns'],
+    'he': ["hurricanedns"],
+    'hedns': ["hurricanedns"],
+    'aurora': ["tldextract", "apache-libcloud"],
+    'acmedns': ["dnspython", ],
+}
```

Comment:
Thanks for separating/gating `aurora` and `acmeDns`, I've been wanting to do that for sometime